### DISCUSSION TITLE
Welcome to WXT's Discussions!

### DISCUSSION THREAD

**Discussion by aklinker1** [wxt-dev/wxt#54] [Announcements]
## 👋 Welcome!

  We’re using Discussions as a place to connect with other members of our community. We hope that you:

  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by chenbimo** [comment]
Your personal web site [aklinker1.io](http://aklinker1.io/) can't access.

**Reply by aklinker1** [reply]
I took it down a while back. Maybe I should redirect it to my github
